sunpy/net/scraper.py

```diff
@@ -81,27 +84,30 @@ class Scraper:
     """
 
     def __init__(self, pattern, regex=False, **kwargs):
```

Comment:
We should remove the regex keyword if we plan for it to do nothing.

Reply:
yes, somehow missed that